wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/WarningLabelView.swift

```diff
@@ -22,8 +22,9 @@ import WireCommonComponents
 
 class WarningLabelView: UIView {
     private let stackView = UIStackView(axis: .horizontal)
-    private let label = DynamicFontLabel(fontSpec: .mediumSemiboldFont, color: SemanticColors.Label.textErrorDefault)
     private let imageView = UIImageView(image: UIImage(named: "Info"))
+    let label = DynamicFontLabel(fontSpec: .mediumSemiboldFont,
```

Comment:
why this label is not private anymore?

Reply:
for testing

Reply:
Maybe it's better to create a public method that will change the value of this label

Reply:
it's value is changed in update(withUser) method. It's public to check it's value

Reply:
We don't usually test the contents of a view by making properties public. Alternatively, you can create a snapshot test.